src/reconcile/rec_write.c

```diff
@@ -4242,15 +4250,18 @@ __rec_col_var(WT_SESSION_IMPL *session,
 
 	/* Set the "last" values to cause failure if they're not set. */
 	last.value = r->last;
-	last.start_ts = last.stop_ts = WT_TS_NONE;
+	last.start_ts = WT_TS_MAX;
+	last.durable_ts = last.stop_ts = WT_TS_NONE;
```

Comment:
This change surprises me a bit. It's a change to the `start_ts`, and in other places `durable_ts` matches `start_ts`, not `stop_ts`. Similarly below.

Reply:
@agorrod, owed you a response:
> This change surprises me a bit. It's a change to the `start_ts`, and in other places `durable_ts` matches `start_ts`, not `stop_ts`. Similarly below.

This is unrelated to the rest of the changes. It's not a bug fix, either, it's just adding some measure of safety. The reason is so we (hopefully) fail if there's a bug and `last.start_ts` isn't set appropriately before being used, that is, a start timestamp of `WT_TS_NONE` is OK, one of `WT_TS_MAX` will fail.